contribs/dvrp/pom.xml

```diff
@@ -66,6 +62,13 @@
 			<groupId>ch.sbb</groupId>
 			<artifactId>matsim-sbb-extensions</artifactId>
```

Comment:
Do you still need that? SwissRailRaptor was migrated to MATSim core. Do you need any of the other parts of the sub-extensions?

Reply:
We are using `LeastCostPathTree` from SBB to compute trees and matrices.